execute/plugin.go

```diff
@@ -292,17 +319,17 @@ func (p *Plugin) Reports(
 	p.lggr.Debugw("transmission schedule override",
 		"transmissionSchedule", transmissionSchedule, "oracleIDToP2PID", p.oracleIDToP2pID)
 
-	report := []ocr3types.ReportPlus[[]byte]{
+	r := []ocr3types.ReportPlus[[]byte]{
 		{
 			ReportWithInfo: ocr3types.ReportWithInfo[[]byte]{
-				Report: encoded,
-				Info:   nil,
```

Comment:
How did we miss this earlier? :D

Reply:
Until now it wasn't needed. A followup PR will decode the info and pass some data into the contract writer.